ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/router_bgp.py

```diff
@@ -291,8 +291,10 @@ def _router_bgp_vlans_vlan(self, vlan, tenant) -> dict | None:
             "struct_cfg": get(vlan, "bgp.structured_config"),
         }
         if self._evpn_gateway_vxlan_l2:
-            bgp_vlan["rd_evpn_domain"] = {"domain": "remote", "rd": vlan_rd}
-            bgp_vlan["route_targets"]["import_export_evpn_domains"] = [{"domain": "remote", "route_target": vlan_rt}]
+            vlan_evpn_domain_scope = get(vlan, "evpn_domain_scope")
```

Comment:
please use `vlan.get("evpn_domain_scope")`. `get()` is only required for deeper lookups, or if we have a required value and need to produce an error. You might also want to add the default value here `vlan.get("evpn_domain_scope", "remote")`

Reply:
Thanks for the tip! It has been changed to the vlan.get() model.